Documentation/gitignore.txt

```diff
@@ -117,7 +117,7 @@ full pathname may have special meaning:
 
  - A leading "`**`" followed by a slash means match in all
    directories. For example, "`**/foo`" matches file or directory
-   "`foo`" anywhere, the same as pattern "`foo`". "`**/foo/bar`"
+   "`foo`" anywhere, the same as pattern "`**/foo/bar`"
```

Comment:
Actually, I think this change is incorrect. `**/foo` has the same effect as `foo`. Not as `**/foo/bar`.

Besides, the Git project asks for your real name and a working email address in the `Signed-off-by:` line: https://git-scm.com/docs/SubmittingPatches#sign-off

Finally, the commit messages in the Git project look very different from the suggested one (which looks more like Subversion style too-short commit messages). In Git's own commit history, the changes need to be [described well](https://git-scm.com/docs/SubmittingPatches#describe-changes).

Reply:
Reading the original sentences again, I think you were startled by the start of the second sentence being `"\`**/foo/bar\`"`... Maybe you would have preferred that pattern to be prefixed by "The pattern", to start the sentence with some English words?

Reply:
Now that I understand `". "` represents the ending of previous sentence, and start of another, I realize my proposed changes are incorrect, though there is certainly opportunity for confusion/improvement.  I consider this an "issue", not worthy of a PR, except it seems PR is necessary to be visible/considered for change via mailing list?

Reply:
>  it seems PR is necessary to be visible/considered for change via mailing list?

No PR is necessary, but it makes it more convenient. As an alternative, you can send the patches directly to the mailing list.